terraform/testcases/otlp_logs/otconfig.tpl

```diff
@@ -0,0 +1,30 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    verbosity: detailed
+  awscloudwatchlogs:
+    log_group_name: "otlp-receiver"
```

Comment:
```suggestion
    log_group_name: "/aws/ecs/otlp/${testingId}"/logs
```

log group can be something like

Reply:
Then we need to change it in CWLogValidator as well. How would I get the testingId there?

Reply:
Suggested those changes